packages/ui/src/theme/css/component/breadcrumbs.scss

```diff
@@ -15,6 +15,20 @@
   line-height: var(--amplify-components-breadcrumbs-item-line-height);
   color: var(--amplify-components-breadcrumbs-item-color);
   font-size: var(--amplify-components-breadcrumbs-item-font-size);
+  &[disabled] {
```

Comment:
nit for consistency, think we use the `&--disabled` convention, is the disabled suffix not getting applied? if not we should fix that